sr/comp/cli/show_schedule.py

```diff
@@ -83,4 +91,8 @@ def add_subparser(subparsers):
         default=15,
         help="how many matches to show (default: %(default)s)",
     )
+    parser.add_argument(
+        '--team',
+        help="restrict to showing matches containing a particular team",
```

Comment:
Perhaps mention here that it's the TLA which should be specified?

Reply:
I've amended the wording and added validation in 8f7a49a83543dfb4e9b1d3447e2399e236e968d1